tools/crm_mon_print.c

```diff
@@ -2085,11 +2085,12 @@ print_stonith_pending(mon_state_t *state, stonith_history_t *history, unsigned i
                 break;
         }
 
+        history = stonith__sort_history(history);
```

Comment:
The two new calls to stonith__sort_history() in crm_mon are unnecessary ... print_stonith_pending() and print_stonith_history() are only called from print_status(), which is only called from mon_refresh_display() after the history has already been sorted. But it shouldn't hurt anything, so we can clean that up later